state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -564,6 +564,7 @@ pub mod pallet {
 			output_asset: Asset,
 			origin: SwapOrigin,
 			request_type: SwapRequestTypeEncoded,
+			broker_fees: Beneficiaries<T::AccountId>,
```

Comment:
just to be safe - we can be sure that the first entry of this array will always be the main broker?

Reply:
Yes, it will always be the main broker